packages/aws-cdk-lib/aws-codebuild/lib/fleet.ts

```diff
@@ -0,0 +1,164 @@
+import { Construct } from 'constructs';
+import { CfnFleet } from './codebuild.generated';
+import { ComputeType } from './compute-type';
+import { EnvironmentType } from './environment-type';
+import { Arn, ArnFormat, IResource, Resource, Token } from '../../core';
+
+/**
+ * Construction properties of a CodeBuild {@link Fleet}.
+ */
+export interface FleetProps {
+  /**
+   * The name of the Fleet.
+   *
+   * @default - CloudFormation generated name
+   */
+  readonly fleetName?: string;
+
+  /**
+   * The number of machines allocated to the compute ﬂeet.
+   * Deﬁnes the number of builds that can run in parallel.
+   *
+   * Minimum value of 1.
+   */
+  readonly baseCapacity: number;
+
+  /**
+   * The instance type of the compute fleet.
+   */
+  readonly computeType: FleetComputeType;
+
+  /**
+   * The environment type of the fleet.
+   */
+  readonly environmentType: EnvironmentType;
+}
+
+/**
+ * Represents a {@link Fleet} for a reserved capacity CodeBuild project.
+ */
+export interface IFleet extends IResource {
+  /**
+   * The ARN of the fleet
```

Comment:
nit: I'd add a period here to stay consistent.

Reply:
Fixed 👍